lib/scss/component/views/stanford_publication.scss

```diff
@@ -97,9 +97,24 @@
 
 .view {
 
+  .su-padding-top-3 {
+    border-bottom: 1px solid $su-color-cool-grey-25;
+
+    &:first-of-type {
+      padding-top: 0;
+    }
+
+    &:last-of-type {
+      border: none;
+      padding-bottom: 0;
+    }
+  }
```

Comment:
this should be nested under a selector of the publication views. as it is now, this would apply styles to all views.